webapi/src/main/scala/org/knora/webapi/messages/StringFormatter.scala

```diff
@@ -142,17 +130,17 @@ object StringFormatter {
   /**
    * The version number of the current version of Knora's ARK URL format.
```

Comment:
```suggestion
   * The version number of the current version of DSP's ARK URL format.
```
Would it be possible to replace all instances of "Knora" with "DSP" here?